### DISCUSSION TITLE
Release v3.3.1

### DISCUSSION THREAD

**Discussion by thomas-bc** [nasa/fprime#2206] [Announcements]
## What's Changed
* The installation guide's instructions have been revised to improve the clarity of virtual environment usage for new users. 

## F´ Tooling changes
* F´ Tools' `new --deployment` now allows for selection of the comDriver type between TcpClient, TcpServer and LinuxUART.
* F´ GDS ships with the UART adapter for all installations.
* F´ GDS is able to run as the TCP client when using the IP adapter, allowing comms with a FSW using TcpServer.

**Full Changelog**: https://github.com/nasa/fprime/compare/v3.3.0...v3.3.1

<hr /><em>This discussion was created from the release <a href='https://github.com/nasa/fprime/releases/tag/v3.3.1'>Release v3.3.1</a>.</em>

**Comment by LeStarch** [comment]
Note: this release required a hot fix: <em><a href='https://github.com/nasa/fprime/releases/tag/v3.3.2'>Release v3.3.2</a>.</em>
